app/services/whatsapp/incoming_message_whatsapp_cloud_service.rb

```diff
@@ -0,0 +1,15 @@
+# https://docs.360dialog.com/whatsapp-api/whatsapp-api/media
+# https://developers.facebook.com/docs/whatsapp/api/media/
+
+class Whatsapp::IncomingMessageWhatsappCloudService < Whatsapp::IncomingMessageBaseService
```

Comment:
Do we need `whatsapp` again in the class name? The class in the module WhatsApp.
`Whatsapp::IncomingMessageCloudService` would suffice.

Reply:
We are trying to set the pattern, IncomingMessage_#{provider_name}. so that all inherited classes will follow a specific naming convention :D